CONTRIBUTING.md

```diff
@@ -26,3 +26,9 @@ information on using pull requests.
 
 This project follows
 [Google's Open Source Community Guidelines](https://opensource.google.com/conduct/).
+
+## Creating a New Layout
+
+If you would like to create a new extension see the [`example`](example) directory. In
```

Comment:
Difficult question but possibly worth clarifying -- Who is responsible for publishing extension to Chrome Web Store?

If I write a layout and this repo accepts my pull request, will chrome-input-extensions publish an extension to the Chrome Web Store? Or should I publish myself?

Reply:
An advantage of "Offered by: chrome-input-extensions" is the extension can be installed in corporate environments with restrictions on extension publishers. (I had to request an exception to install https://chrome.google.com/webstore/detail/uk-colemak/nionfllpgckhdmcecikpfkonedlmlnop at work)

Reply:
Curiously both https://chrome.google.com/webstore/detail/svorak/ijimhcgeahpgfdcgaheadagkjkiibcnj and https://chrome.google.com/webstore/detail/lushootseed/bdcecklhaeiniooomgajkefmnghopnpk  are "Offered by: chrome-input-extensions"  but only the former is labelled ["By Google"](https://user-images.githubusercontent.com/105314/115610460-2e221f80-a2e9-11eb-9ca6-bc8b1b253a89.png).